### ISSUE TITLE
Feature request: Manually trigger circuit breaker rollback

### ISSUE THREAD

**Issue by oVertoneFenton** [aws/copilot-cli#2041]
`copilot version: v1.2.0`
I'm attempting to deploy a service that fails repeatedly due to a networking mistake on my part. The service successfully starts a task, but the task is unhealthy (sends 503 because of failed database connection).
<img width="375" alt="Screen Shot 2021-03-10 at 1 58 20 PM" src="https://user-images.githubusercontent.com/61425462/110697122-22e7ba00-81a9-11eb-95cf-31f57af37302.png">

The problem is that any number of retries will not fix the underlying networking issue, therefore I have to wait 30+ minutes for the task to fail 10 times before circuit breaker triggers a rollback. It would drastically speed up development to be able to exit the process immediately and start again without having to wait for 10 failures.

Please add a feature that can interrupt the deploy process and trigger a circuit breaker rollback. Something like:
```
copilot svc deploy -n my-service -e test --trigger-rollback
```

**Comment by efekarakus** [comment]
Thanks @oVertoneFenton ! the feature request makes sense to me!

**Comment by paulca** [comment]
This would _really_ help. Because of how black box some of the deployment to ECS seems, having to wait for 10 failures before tweaking a setting and trying again feels like a lot.

**Comment by DaveCollinsJr** [comment]
Agreed!  When testing something that works fine in local development, but fails when deployed to ECS using copilot (e.g. Asset compilation issues, webpacker config), you can only test once every 30 minutes or so.  Really slows down the try-deploy-tryagain cycle! 😆

**Comment by KollaAdithya** [comment]
Hello @paulca @DaveCollinsJr @oVertoneFenton !
Ctrl-C feature is released in [v1.30](https://github.com/aws/copilot-cli/releases/tag/v1.30.0) 🎉 . You can now cancel your stack update to rollback to previous configuration.
